src/create-support-ticket.ts

```diff
@@ -10,59 +9,69 @@ import {
 import {
   CreateTicketPayload,
   HttpError,
-  SuspiciousActivityEvent,
+  ReportSuspiciousActivityEvent,
 } from "./common/model";
-import { CREATE_TICKET_PATH } from "./common/constants";
-
-export const formatCommentBody = (event: SuspiciousActivityEvent): string => {
+import {
+  CREATE_TICKET_PATH,
+  SUSPICIOUS_ACTIVITY_EVENT_NAME,
+} from "./common/constants";
+import { DynamoDBDocumentClient, UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+
+const dynamoClient = new DynamoDBClient({});
+const dynamoDocClient = DynamoDBDocumentClient.from(dynamoClient);
+
+export const formatCommentBody = (
+  event: ReportSuspiciousActivityEvent
+): string => {
   const htmlBody = [];
-  const eventDateTime = new Date(event.timestamp * 1000);
 
-  htmlBody.push(`<p><strong>Event Name</strong>: ${event.event_name}</p>`);
-
-  if (event.event_id) {
-    htmlBody.push(`<p><strong>Event ID</strong>: ${event.event_id}</p>`);
-  }
+  htmlBody.push(
+    `<p><strong>Event Name</strong>: ${event.suspicious_activity.event_type}</p>`
+  );
 
-  if (event.client_id) {
-    htmlBody.push(`<p><strong>Client ID</strong>: ${event.client_id}</p>`);
+  if (event.suspicious_activity.event_id) {
+    htmlBody.push(
+      `<p><strong>Event ID</strong>: ${event.suspicious_activity.event_id}</p>`
+    );
   }
 
-  if (event.timestamp) {
+  if (event.suspicious_activity.timestamp) {
     htmlBody.push(
-      `<p><strong>Date and Time</strong>: ${eventDateTime.toUTCString()}</p>`
+      `<p><strong>Reported Date and Time</strong>: ${new Date(
+        event.suspicious_activity.timestamp
+      ).toUTCString()}</p>`
     );
   }
 
-  if (event.user?.user_id) {
-    htmlBody.push(`<p><strong>User ID</strong>: ${event.user.user_id}</p>`);
+  if (event.suspicious_activity.client_id) {
+    htmlBody.push(
+      `<p><strong>Client ID</strong>: ${event.suspicious_activity.client_id}</p>`
+    );
   }
 
-  if (event.user?.session_id) {
+  if (event.suspicious_activity.user_id) {
```

Comment:
should we use optional chaining here? It's used on line 59

Reply:
Removed the optional on line 59 because the suspicious activity field is mandatory is a mandatory field in the model